### PR TITLE
chore: Update changelog for v8 final release

### PR DESCRIPTION
**Pull request by glnro**

### CODE REVIEW

**Review by mpoke — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -94,60 +61,6 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (tests) Add [E2E ccv tests](https://github.com/cosmos/gaia/blob/main/tests/e2e/e2e_gov_test.go#L138). Tests covering new functionality introduced by the provider module to add and remove a consumer chain via governance proposal.
 * (tests) Add [integration ccv tests](https://github.com/cosmos/gaia/blob/main/tests/ics/interchain_security_test.go). Imports Interchain-Security's `TestCCVTestSuite` and implements Gaia as the provider chain.
 
-## [v9.0.0-rc0] - 2023-01-19
```
Comment (shaspitz) [inline CHANGELOG.md]: Did you mean to remove this section in regard to v9?

Comment (glnro) [reply CHANGELOG.md]: nope, eventually yes but not for now

**Review by shaspitz — APPROVED** [review]
LGTM

**Review by jtremback — COMMENTED** [review]
LGTM

**Pull request merged**